api/wasm/cpp/proxy_wasm_intrinsics_lite.proto

```diff
@@ -6,3 +6,81 @@ import "struct_lite.proto";
 message WKT {
   google.protobuf.Struct struct = 1;
 }
+
+message DataSource {
```

Comment:
First, these have to have the "lite" annotation, second the ones in envoy have lots of includes which would also have to be pulled in and finally I don't know how to get bazel to do what you are suggesting.  I think it is a very good idea and we should do it as part of upstreaming unless you know a relatively easy way to do it now in which case I am all ears.

Reply:
Ah, I've missed the "lite" annotation, this might be tricky indeed.

As for Bazel - we'd just need to split some messages upstream, so that we don't pull too much unnecessary stuff, and then we can just include those protos, but that might be better suited for the upstreaming work.

Let's not block this PR on this.